Task-3/Task-3/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace Task_3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int inputData;
+            bool success = int.TryParse(Console.ReadLine(), out inputData);
+
+            if (success)
+            {
+                Console.WriteLine("input data: " + inputData);
+            }
+            else
+            {
+                Console.WriteLine("Нou need to enter a number!");
+            }
+            Console.ReadKey();
+
```

Comment:
Add whiteline above Console.ReadKey and revome below ReadKey.

Reply:
Changed